internal/pyxis/pyxis_suite_test.go

```diff
@@ -157,6 +157,12 @@ func pyxisTestResultsHandler(ctx context.Context) http.HandlerFunc {
 		switch {
 		case request.Header["X-Api-Key"][0] == "my-bad-testresults-api-token":
 			response.WriteHeader(http.StatusUnauthorized)
+		case request.Header["X-Api-Key"][0] == "my-unrecognized-perflight-version":
```

Comment:
Are you misspelling `preflight` on purpose?

Reply:
Good catch, this was _not_ on purpose.